vm-aws-yaml/Pulumi.yaml

```
@@ -0,0 +1,124 @@
+name: vm-aws-yaml
+description: A YAML program to create a virtual machine on AWS
+runtime: yaml
+
+template: 
+    description: A YAML program to create a virtual machine on AWS
```

I think we wanted to remove the language names for these, right? 

```suggestion
    description: A virtual machine on Amazon EC2
```